### PR TITLE
Fix snack.expo.io

### LINKED ISSUE #3140: [Broken Website] Expo Snack QR code
**Website**
https://snack.expo.io/
(used by react native official docs)

**How to Reproduce**
An example of this is
> 1. Go to https://snack.expo.io/
> 2. Make sure the "My device | iOS | Android | Web" button group is set to "My device"
> 3. Try scanning the QR code with the [Expo mobile app](https://play.google.com/store/apps/details?id=host.exp.exponent)

**Expected behavior**
The QR code scans properly. The white border around the QR code is required for this to work.

**Actual behavior**
The Expo app is not even aware there is a QR code. This is because Dark Reader darkened the white border, breaking the QR code.

**Screenshots**

image
Screenshot_20200718-192822

**System info:**
 - OS: `Ubuntu 20.04 Desktop`
 - Browser: `Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.89 Safari/537.36`
 - Darkreader Version: `4.9.15`

**Additional Context**
- Using Expo app `2.16.1` on Android 10
- The QR code is drawn on a `<canvas>` with the position markers flush to the edges
- The 2nd level of wrapping `<div>` has a `4px` rounded border, and that needs to stay white

### PR DESCRIPTION
**Pull request by Simsimpicpic**
QR Code is not readable.
> **Before** Screenshot_2020-07-19 awkward celery

> **After** Screenshot_2020-07-19 Snack - React Native in the browser

This should fix #3140.

### CODE REVIEW

**Comment by Gusted** [comment]
We don't have suchs NO INVERT on dynamic theme. That's why the build is failing. I had a little research on the issue already it should be easy fixed, but just like with whatsapp with their we code their classes where changes every 3 weeks. I thought of a possible solution but I should have a better look tomorrow.

**Comment by Simsimpicpic** [comment]
Ok 👍🏻
(i don't know why i put "no invert" 😅)

**Comment by Gusted** [comment]
Btw as their is no use in using Invert of this you can change it to
```
CSS
the Selector {
    background-color: white !important;
} 
```
I will have a look for a universal Selector that isn't based of a class name that will probably change in the next few weeks.

**Comment by Gusted** [comment]
NO INVERT is for Filter mode ;)

**Comment by Simsimpicpic** [comment]
Ok 😁

**Comment by Gusted** [comment]
Btw if you want give it a try.
I probably go with Child Combiner https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator
that starts at a normalized class(e.g. For this site I saw something like id="root") so `#root > div > div > div:nth-child(1)` will it probably look like.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4029,6 +4029,15 @@ CSS
 
 ================================
 
+snack.expo.io
+
+CSS
+._1rs24yq {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4035]: `#root > div > div > div:nth-child(2) > div:nth-child(2) > div:nth-child(2) > div > div:nth-child(2)` Their you go

**Pull request merged**